### PR TITLE
Add sts service call credential type to sd tracer

### PR DESCRIPTION
**Pull request by bianpengyuan**

### CODE REVIEW

**Comment by bianpengyuan** [comment]
xref of envoy pr that adds the config: https://github.com/istio/envoy/pull/151

**Review by elfinhe — APPROVED** [review]

File: tools/packaging/common/envoy_bootstrap_v2.json [hunk 1]
```diff
@@ -639,6 +639,28 @@
...
+                "filename": "/etc/ssl/certs/ca-certificates.crt"
+              }
+            }
+          },
+          "call_credentials": {
+            "sts_service": {
+              "token_exchange_service_uri": "http://localhost:{{ .stsPort }}/token",
+              "subject_token_path": "/var/run/secrets/tokens/istio-token",
+              "subject_token_type": "urn:ietf:params:oauth:token-type:jwt",
+              "scope": "https://www.googleapis.com/auth/cloud-platform",
```
Comment (howardjohn) [inline tools/packaging/common/envoy_bootstrap_v2.json]: why is google stuff here?

Comment (bianpengyuan) [reply tools/packaging/common/envoy_bootstrap_v2.json]: This is a field in stackdriver tracer config, which is google stuff.

**Comment by bianpengyuan** [comment]
@JimmyCYJ could you also take a look? thanks!

**Review by howardjohn — APPROVED** [review]

**Pull request merged**